flashcards/backend/docs/config.rst

```diff
@@ -0,0 +1,53 @@
+config package
```

Comment:
We are not documenting config module for now. Remove this file please 😅. To avoid generating rst files for random packages, from inside the container you can run `sphinx-apidoc -o docs decs` we have to be explicit for generating docs

Reply:
@bartoszbaran2 after reseting branch i thing you still have this file

Reply:
deleted